contents/uix/uix.md

```diff
@@ -0,0 +1,47 @@
+
+Learning the key differences between UI and UX can not only help us identify and settle down for the correct job role, but can also help us apply each of these concepts appropriately to our own applications. This document will address the difference between UI and UX as well as some of the techniques used in UI Design and UX Design.
+
+## User Interface vs User Experience
+
+UI is about the design of the buttons, the layout of the website and the responsiveness of the application. It encompasses the presentation and interactivity of the application. On the other hand, UX is concern with the ease of use and the pleasure provided through using the application. In short, UI is about the look and feel while UX is about customer satisfaction.
+
+Due to the overlapping nature of UI and UX, you might still have some questions such as *'Doesn't improving the UI also improve the UX?'* Indeed, it does. In fact, UI **can** be part of UX.
+
+> Note that I used **can** because UI is not longer a necessity to UX due to the emergence of [zero UI](http://blog.careerfoundry.com/ui-design/what-is-zero-ui), which won't be covered here.
+
+To achieve a better understanding, let's look at real life examples of good UI but bad UX. A prime example would be the [Mystery Meat Navigation](https://www.techinasia.com/talk/material-design-mystery-meat-navigation-problem) problem. It refers to buttons and links that do not explain themselves; users have to click on them to find out where it leads. In essence, the pursuit of clean and neat UI has resulted in omission of details that provide clarity for the user. This results in bad UX because people do not like to puzzle over how things work. A good UX should never make the user think.
+
+If you would like to read more about the differences between UI and UX, here are a few articles:
+
+* [The Difference Between UX and UI Design-A Layman’s Guide](http://blog.careerfoundry.com/ui-design/the-difference-between-ux-and-ui-design-a-laymans-guide/) - provides breakdown of the disciplines involved.
+* [What’s the difference between UI and UX?](https://www.usertesting.com/blog/2016/04/27/ui-vs-ux/) - provides opinions from UX experts.
+* [The superpowers of UX & UI designers](https://www.ymedialabs.com/ux-vs-ui/) - provides illustrations to highlight the differences.
+
+## UI Design
+
+UI Design is highly dependent on your sense of visual design, but thanks to the rise of flat and minimalist design you don't have to be a photoshop expert to get started on UI design. 
+
+A good way to start is by observing existing designs. Use existing UI Designs as reference and try to apply the design patterns on your own application. Using a [UI design guidelines](http://www.goodui.org/) can help to highlight and avoid common mistakes. Online lessons like [Hack Design](https://hackdesign.org/lessons#graphic-design-principles) also provides a variety of materials covering the basics of UI design. The best thing is that it is free!
```

Comment:
"...as references...apply the design patterns in your..."
"Using UI design guidelines..."
"Online lessons...also provide..."
"...they are free!"